### PR TITLE
Add blog, update materials

### PR DESCRIPTION
**Pull request by skaltman**
Blog preview here: https://connect.posit.it/brand-yml-blog/

### CODE REVIEW

File: blog.qmd [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+* **Team consistency**: Ensuring all contributors follow branding guidelines across reports, dashboards, and apps can be challenging to manage. 
+
+### The solution: **brand.yml**
+
+Enter [**brand.yml**](https://posit-dev.github.io/brand-yml/). We created **brand.yml** to simplify the process of applying branding guidelines to your work. With **brand.yml**, you can define your organization's branding in a single YAML file and apply that branding across reports, dashboards, and apps created with Posit’s open-source tools.
+
+### Advantages of **brand.yml**
+
+**brand.yml** allows you to collect a brand's logos, colors, fonts, and typographic choices into a single `_brand.yml` file, which you can then apply instantly across Quarto reports and Shiny apps.
```
Comment (gadenbuie) [inline blog.qmd:28]: > Quarto reports, Shiny apps, and more.

I think it’s good to call out early that it’s not just for Quarto and Shiny.

File: blog.qmd [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+## See **brand.yml** in action
+
+Let's see how **brand.yml** works. The first step is to define your brand elements, including logos, colors, fonts, and typography, in a single `_brand.yml` file. 
+
+Below is an example `_brand.yml` file that defines a logo, color palette, and font and typographic choices for an example brand. Learn more about the structure of a `_brand.yml` file [here](https://posit-dev.github.io/brand-yml/brand/).
+
+
+```yaml
+logo:
```
Comment (gadenbuie) [inline blog.qmd:46]: I’m partial to including `meta` here, even if it’s just `meta.name`.

File: blog.qmd (hunk 1 above)
Comment (skaltman) [inline blog.qmd:28]:
```suggestion
**brand.yml** allows you to collect a brand's logos, colors, fonts, and typographic choices into a single `_brand.yml` file, which you can then apply instantly across Quarto reports, Shiny apps, and more.
```

File: blog.qmd (hunk 2 above)
Comment (skaltman) [reply blog.qmd:51]: done! Thanks @gadenbuie

**Pull request merged**